amqp-impl/src/main/java/io/streamnative/pulsar/handlers/amqp/AbstractAmqpExchange.java

```diff
@@ -30,6 +31,8 @@ public abstract class AbstractAmqpExchange implements AmqpExchange {
     protected boolean autoDelete;
     protected boolean internal;
     protected Map<String, Object> arguments;
+    protected Map<String, Set<AmqpQueue>> bindingKeyQueueMap;
```

Comment:
When to remove the items from this map?

Reply:
The item will be removed from the map, when the queue is unbound with the exchange.

Reply:
Improve the cleanup logic here https://github.com/streamnative/aop/pull/620/files#diff-9ab7ce94c7044f408229e43516108042b0ed97a47c5f1eea44c2ac2e4a0fa3e5R251, PTAL